R/estimate_infections.R

```diff
@@ -63,22 +63,26 @@
 #' reported_cases <- example_confirmed[1:60]
 #'
 #' # set up example generation time
-#' generation_time <- generation_time_opts(
+#' generation_time <- get_generation_time(
 #'   disease = "SARS-CoV-2", source = "ganyani", fixed = TRUE
 #' )
 #' # set delays between infection and case report
 #' incubation_period <- get_incubation_period(
+#'   disease = "SARS-CoV-2", source = "lauer", fixed = TRUE
+#' )
+#' # delays between infection and case report, with uncertainty
+#' incubation_period_uncertain <- get_incubation_period(
 #'   disease = "SARS-CoV-2", source = "lauer"
 #' )
-#' reporting_delay <- list(
+#' reporting_delay <- dist_spec(
 #'   mean = convert_to_logmean(2, 1), mean_sd = 0,
```

Comment:
if you have changed mean and sd to be on the natural scale then this is likely wrong.

Reply:
For now they're unchanged but I think we should rename them logmean/logsd or mu/sigma here (with mean/sd acquiring their natural meanings).

Reply:
very much agree on renaming and agree we should/can support the natural scale input with an internal transform. I think we should doc that that isn't an ideal approach though.

Reply:
This would potentially require a larger rethink - see #382.